examples/opf/experiments/anomaly/temporal/hotgym/description.py

```diff
@@ -317,6 +313,8 @@
         'anomalyParams': {
            'mode': 'likelihood', # pure(=default) / weighted / likelihood
            'slidingWindowSize': 5, # >=0 / None
+           'binaryAnomalyThreshold': None, # None or from (0,1)
+           'computeFn': Anomaly.COMPUTE_RAW,
```

Comment:
ok, then I'll just remove it here, in some other PR, I have a new "anomaly-swarming" example, so I'll stuff all the experimental anomaly functionality there.